.infrastructure/README.md

```diff
@@ -0,0 +1,101 @@
+## Requirements
+
+The following tools are required to setup and manage a new environment. 
+
+1. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) installed.
+2. [tfenv](https://github.com/tfutils/tfenv) to mange terraform versions.
+
+## How setup a new environment
+
+Make sure you don't have a `*.tfstate` file within the `.infrastructure/.terraform` folder. If you have any, delete them.
+
+### Step 1: Disable the backend
+
+Comment the `backend "s3" {}` line from `00-main.tf` file:
+
+``` sh
+terraform {
+  required_version = "1.2.8"
```

Comment:
If you update the version of the `00_main.tf` file, I'd update also here in the documentation

Reply:
Correct